tam/week1/day1/main.go

```diff
@@ -0,0 +1,31 @@
+package main
+
+import (
+	"fmt"
+)
+
+func fib(n int) int {
+	fmt.Printf("Finding the Fibonacci number at the position %d ...\n", n)
+	defer fmt.Println("Done!")
+	if n <= 0 {
+		fmt.Println("Invalid input! Please enter a positive integer.")
+		return -1
+	}
```

Comment:
```suggestion
	if n < 0 {
		fmt.Println("Invalid input! Please enter a positive integer.")
		return -1
	}
```
The position `0` is valid.
Ex:
```sh
Fib: 0, 1, 1, 2, 3, 5, 8, 13, 21, ...
Output:
   - Index: 0 - Output: 0
   - Index: 1 - Output: 1
   - Index: 3 - Output: 2
```